storage/src/journal/fixed.rs

```diff
@@ -682,7 +682,7 @@ impl<E: Storage + Metrics, A: Codec<Cfg = ()> + FixedSize> Journal<E, A> {
 
     /// Safely removes any previously tracked blob from underlying storage.
     async fn remove_blob(&mut self, index: u64, blob: Append<E::Blob>) -> Result<(), Error> {
-        blob.close().await?;
+        blob.sync().await?;
```

Comment:
Same question as above regarding whether we can just drop the blob and silently discard any unsynced data.